<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtPing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>625</width>
     <height>275</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping host:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>10</y>
      <width>290</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>www.google.cat</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>601</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="copyButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doPingButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPackets">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>50</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tries:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">10</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQtping2">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuQtping2"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
